Source/WebKit/WebProcess/WebPage/WebPage.h

```diff
@@ -2600,6 +2600,7 @@ class WebPage : public API::ObjectImpl<API::Object::Type::BundlePage>, public IP
     };
     std::optional<DeferredDidReceiveMouseEvent> m_deferredDidReceiveMouseEvent;
 
+    HashSet<WebCore::ResourceLoaderIdentifier> m_networkResourceRequestIdentifiersForPageLoadTiming;
```

Comment:
Maybe a separate PR, but this and m_trackedNetworkResourceRequestIdentifiers could be abstracted into an object that in release builds just increment and decrement an integer and in debug builds maintains a set to assert that we aren't re-adding an existing one.